js/main.js

```diff
@@ -1,20 +1,57 @@
-function getRandomIntInclusive (min, max) {
-  let result;
-  if (min < 0 || max < 0 || max <= min) {
-    result = 'Проверьте, пожалуйста, введенный диапазон чисел: диапазон может быть только положительный, включая ноль, а максимальное значение должно превышать минимальное.';
-  } else {
-    min = Math.ceil(min);
-    max = Math.floor(max);
-    result = Math.floor(Math.random() * (max - min + 1)) + min;
-  }
-  return result;
+const COMMENT_DESCRIPTIONS = ['Пляж отеля', 'Go to the beach', 'Цвет моря', 'Подруга', 'Супчик', 'Макларен', 'Десерт', 'Вкусный напиток', 'Летают самолеты', 'Храним обувь', 'Путь к морю', 'Ауди', 'Ужин', 'Пункт меню', 'Отдыхаю', 'Лечу в отпуск', 'Концерт', 'Ретро-машина', 'Тапочки с подсветкой', 'Пальмы', 'Вилка', 'Закат', 'Краб', 'Руки на концерте', 'Бегемот',];
+const COMMENT_NAMES = ['Дарья', 'Алексей', 'Иван', 'Наталья', 'Юлия', 'Андрей',];
+const MESSAGES = ['Всё отлично!', 'В целом всё неплохо. Но не всё.', 'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.', 'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.', 'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.', 'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!',];
+const SIMILAR_DESCRIPTIONS_COUNT = 25;
+const MAX_COMMENT_COUNT = 4;
+const idMessageArray = [];
+const idDescriptionArray = [];
+//объявляем функцию, генерирующую случайные числа из заданного диапазона
+const getRandomPositiveInteger = (a, b) => {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+//объявляем функцию, выбирающую случайный элемент заданного массива
+const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length-1)];
+//объявляем функцию, формирующую коммент из  одного или двух случайных значений массива MESSAGES
+const getRandomMessage = () => getRandomPositiveInteger(1, 2) < 2 ? getRandomArrayElement(MESSAGES) : `${getRandomArrayElement(MESSAGES)  } ${  getRandomArrayElement(MESSAGES)}`;
+//объявляем функцию, генерирующую случайный неповторяющийся id Message
+function getRandomIdMessage() {
+  let idM = getRandomPositiveInteger(1, 10**10);
```

Comment:
```suggestion
  let idM = getRandomPositiveInteger(1, 10 ** 10);
```

а зачем 10 в 10 степени?

Reply:
Экономил ресурс ночью и решил с запасом сделать, что было с моей стороны некорректно. Вот так записал сейчас:
let idM = getRandomPositiveInteger(1, MAX_COMMENT_COUNT*SIMILAR_DESCRIPTIONS_COUNT);